README.md

```diff
@@ -42,6 +42,7 @@ This configuration should work for a default TorizonCore image:
 ```
 [torizon]
 url = "https://dgw.torizon.io/ras/"
+director_url = "https://dgw.torizon.io/director/"
```

Comment:
I am not sure about adding this here. I just added this config option for testing and running rac locally/development. If director_url is not present RAC will derive director url from `url`, so this is not mandatory.. unless you have a good reason to make this explicit?